wcivf/apps/elections/views/mixins.py

```diff
@@ -202,6 +204,49 @@ def get_global_registration_card(self, post_elections):
             "election_date": next_ballot.election.election_date,
         }
 
+    def get_global_postal_vote_card(self, post_elections, council):
+        if not post_elections:
+            return {"show": False}
+        next_ballot = post_elections[0]
+        election = next_ballot.election
+        country = next_ballot.post.territory
+
+        if not country:
+            country = Country.ENGLAND
+        else:
+            country = {
+                "ENG": Country.ENGLAND,
+                "SCT": Country.SCOTLAND,
+                "WLS": Country.WALES,
+                "NIR": Country.NORTHERN_IRELAND,
+            }.get(country)
+        election = from_election_id(election_id=election.slug, country=country)
```

Comment:
There is a chunk of code here I pretty much copy/pasted from `get_global_postal_vote_card()`.
We could probably extract some of this and DRY up, but for the moment I've just duplicated it.
While our view on this is "this code is experimental and we might throw it away", I don't want to optimise it too much.